src/pages/stops/stops.html

```diff
@@ -14,8 +14,8 @@
 
 <ion-content>
   <ion-list>
-    <button ion-item *ngFor="let item of items" (click)="itemTapped($event, item)">
-      {{item.title}}
+    <button ion-item *ngFor="let item of items | async" (click)="itemTapped($event, item)">
+      {{item.name}}:{{item.lineShort}}
```

Comment:
We don't really need the line here either (it will complicate things since a stop can have multiple lines)

Reply:
agreed